README.md

```diff
@@ -1,24 +1,183 @@
-# README
+<a name="readme-top"></a>
+<!-- TABLE OF CONTENTS -->
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Ruby version
+<!-- PROJECT DESCRIPTION -->
 
-* System dependencies
+# 📖 Budget_App <a name="about-project"></a>
 
-* Configuration
+**Budget_App**
+is a mobile web application designed for budget management. Easily track your expenses by organizing transactions into categories, enabling you to monitor your spending and its allocation.
 
-* Database creation
 
-* Database initialization
+## 🛠 Built With <a name="built-with"></a>
 
-* How to run the test suite
+### Tech Stack <a name="tech-stack"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby on rails</a></li>
+  </ul>
+</details>
 
-* Deployment instructions
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">postgres</a></li>
+  </ul>
+</details>
 
-* ...
+<details>
+<summary>Bootstrap5</summary>
+  <ul>
+    <li><a href="https://getbootstrap.com/docs/5.0/getting-started/introduction/">Bootstrap5</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- Ruby
+- Rails
+- PostgreSQL (RDB)
+- Bootstrap5
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://budget-denc.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Walkthrough Demo <a name="live-demo"></a>
+
+- [Walkthrough Demo Link](https://www.loom.com/share/ef3bca64baed405c8d1fa14ad5c333ec?sid=64a81412-87d0-4e2d-8779-4a6e0f03dc11)
+
```

Comment:
- [ ] **Good presentation so far 👏 👏** However, according to the project requirements, I think it would be better to highlight a piece of the code that is more interesting or proud of making 🙏

Reply:
What if I'm proud of the entire code I've written? Can't I say that in the video?